src/lpwFuncs.cpp

```diff
@@ -250,6 +251,7 @@ void userActivity() { // Call of this function can be found in line 181, buttons
 }
 
 void initDormant() {
+  goingDormant = true;
   delay(2000);
```

Comment:
can we remove this delay?
(initially we had a 2 sec delay to be able to read the "byeBye"-message on the display. 
I think it makes more sense to signal the sleep mode using a tone sequence (maybe the startup tones reversed?)